components/automate-cluster-ctl/spec/automate_cluster_spec.rb

```diff
@@ -0,0 +1,24 @@
+#
+# Author:: Faizan Fulara (<ffulara@progress.com>)
```

Comment:
Why is this header added in some files and not in some files ?

Reply:
Added header with Apache license in ruby files to avoid any dependency license thing